packages/datetime/src/components/date-range-picker/_daterangepicker.scss

```diff
@@ -115,6 +115,11 @@
   justify-content: space-around;
   width: 100%;
 
+  &.#{$ns}-daterangepicker-timepickers-stacked {
```

Comment:
also, since this class is only being applied in @blueprintjs/datetime2 components, I think the new styles belong in that package too.

I can push a commit to finish up the PR if you don't mind?

Reply:
Ah, makes sense. Feel free to push fixes to these issues!